### ISSUE TITLE
Feature request:  Simpler view generation?

### ISSUE THREAD

**Issue by snhirsch** [pgmodeler/pgmodeler#1081]
I am finding pgModeler to be very valuable indeed for my job as data architect and wanted to express my appreciation for your work.  However, I'm finding the view creation process to be extremely (and, in my opinion, unnecessarily) painful.  Can you not take better advantage of Qt and support clicking on the tables and columns you wish to participate in the view?  

At a minimum, I'm finding the behavior of the 'table' and 'column' fields to be very cumbersome.  Once you have drilled through and defined the table (possibly with alias) for a SELECT clause, why does the entire process have to start over from the top for the column in that same table?  

If none of this is practical, then at least allow me to define the view in one shot by importing an SQL file.

**Comment by rkhaotix** [comment]
I recognize that the view creation is a bit complicated and I'm planning to work on the UX of the entire tool when starting to develop pgModeler 1.0. The view creation will be the first one that I'll improve.

About importing an SQL file and use its contents as source of the view... This is already possible (see the image). To do that, check ```Reference type``` as ```Expression``` and then check the box ```View Definition```. After that you can create the full definition of the view by hand or loading it from an external file.

![view_form](https://user-images.githubusercontent.com/2205476/37221406-9426e2f2-23a8-11e8-918a-1983e0dcd4ea.png)

**Comment by snhirsch** [comment]
Thanks, I gave that a try, but the result does not show the columns in the view, so not useful to me.  Also, pgModeler threw an error box when I tried to save the design.  Was not able to proceed without deleting the view.  At this point, I'm finding it quicker and simpler to simply code the views directly in XML and paste into the dbm file.  I'll write a little Perl or Python script to help me generate XML in the future.
